go.mod

```
@@ -4,13 +4,12 @@ go 1.17
 
 require (
 	github.com/cosmos/cosmos-sdk v0.42.9
-	github.com/forbole/juno/v2 v2.0.0-20220223115557-e4ed06a008de
+	github.com/forbole/juno/v3 v3.0.0-20220315104134-6cea725e4656
```

should we use the latest juno version? 
```suggestion
	github.com/forbole/juno/v3 v3.0.0-20220316215646-3b5d944a0def
```